clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/remote/InvalidationNearCacheResourceDefinition.java

```diff
@@ -67,7 +71,17 @@ public AttributeDefinition getDefinition() {
     }
 
     static void buildTransformation(ModelVersion version, ResourceTransformationDescriptionBuilder parent) {
-        // No transformations yet
+        ResourceTransformationDescriptionBuilder builder = parent.addChildResource(PATH);
+        if (InfinispanModel.VERSION_11_0_0.requiresTransformation(version)) {
+            builder.getAttributeBuilder().setValueConverter(new AttributeConverter.DefaultAttributeConverter() {
```

Comment:
Should we be converting or rejecting the defined attribute?

Reply:
The previous behavior was equivalent to a value of FALSE.  Thus, if the value is undefined or FALSE, then we discard.  Otherwise (i.e. it is defined and TRUE), then we reject.  Is this incorrect?

Reply:
If it defined, we leave it defined.  We only need to convert the case where it is now undefined (because it was required previously).  Is that incorrect?

Reply:
Ah okay so it's no longer required? If that's the case this seems okay.

Reply:
Sorry.  My mistake. It was *not* required previously, but it had a weird default value, which is better expressed as being undefined.  So, the conversion is: if undefined, set it to the old default value.  If defined, leave as defined.  Does that make sense?